tests/uptane_vector_tests.cc

```diff
@@ -83,6 +84,7 @@ bool run_test(Json::Value vector) {
 }
 
 int main(int argc, char* argv[]) {
+  ::testing::InitGoogleTest(&argc, argv);
```

Comment:
clion was passing some random gtest parameters to the test class, so adding this caused them to be passed to GoogleTest (and removed from argv) rather than causing the test to fail